app/views/content/is-teaching-right-for-me/engineers-teach-physics/_article.html.erb

```diff
@@ -34,9 +34,9 @@
     <%= render TextBlockWithTwoImagesComponent.new(
       heading: "Engineering the future",
       colour: "pink",
-      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_path: "static/images/content/hero-images/teacher2.jpg",
       image_1_alt: "",
-      image_2_path: "static/images/content/hero-images/0004.jpg",
+      image_2_path: "static/images/content/hero-images/physics.jpg",
       image_2_alt: "",
     ) do %>
       <p>
```

Comment:
have updated, but can't delete this image, its one of those that used on the how to become a teacher page

Reply:
Is this the one in the structured how to data? If so, that will need replacing anyway.....

We should discuss this with Martyn and Sarah - this structured how to data isn't actually live - from an SEO perspective, we might want to pick this up again and get it live, otherwise we should remove this altogether (so you don't have image issues)